File: snippet/query-examples/csharp/har-get.cs
var client = new RestClient("https://will-x86.com/api/pixelsq?x=1&y=2&hex=%23102932");
var request = new RestRequest(Method.GET);
request.AddHeader("User-Agent", "curl/7.68.0");
request.AddHeader("Accept", "*/*");
IRestResponse response = client.Execute(request);

File: snippet/query-examples/kotlin/har-get.kt
val client = OkHttpClient()

val request = Request.Builder()
  .url("https://will-x86.com/api/pixelsq?x=1&y=2&hex=%23102932")
  .get()
  .addHeader("User-Agent", "curl/7.68.0")
  .addHeader("Accept", "*/*")
  .build()

val response = client.newCall(request).execute()
